### PR TITLE
Fixes #830 - normalizes gulp task names + gulp cleanup

### LINKED ISSUE #830: Is gulp-load-plugins using `$` a convention?
`gulp-load-plugins` is mascarading as jquery in [our gulp tasks](https://github.com/cfpb/cfgov-refresh/blob/flapjack/gulp/tasks/test.js#L4). Is this a convention? It doesn't appear to be according to the plugin [GH repo](https://github.com/jackfranklin/gulp-load-plugins#usage).

### PR DESCRIPTION
**Pull request by anselmbradford**
Fixes #830 - normalizes gulp task names + gulp cleanup

## Changes

- Dash cases filenames, per nodejs conventions.
- Updates ambiguous `$` to `plugins`.
- Narrows specificity of `config` name.

## Testing

- gulp tasks should work as before.

## Review

- @sebworks 
- @KimberlyMunoz 
- @jimmynotjim

### CODE REVIEW

File: gulp/tasks/copy.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 'use strict';
 
 var gulp = require( 'gulp' );
-var $ = require( 'gulp-load-plugins' )();
-var config = require( '../config' ).copy;
-var handleErrors = require( '../utils/handleErrors' );
+var plugins = require( 'gulp-load-plugins' )();
```
Comment (sebworks) [inline gulp/tasks/copy.js]: Should this be `gulpLoadPlugins`?

Comment (anselmbradford) [inline gulp/tasks/copy.js]: I would get behind `gulpLoadPlugins` too, as it describes the package name better.. but the original issue said `plugins` and that's at least clearer than `$`.

File: gulp/tasks/copy.js [hunk 2]
```diff
@@ -23,33 +23,38 @@ function _genericCopy( src, dest ) {
...
 
 gulp.task( 'copy:icons', function() {
-  return _genericCopy( config.icons.src, config.icons.dest );
+  var icons = configCopy.icons;
+  return _genericCopy( icons.src, icons.dest );
 } );
 
 gulp.task( 'copy:vendorfonts', function() {
-  return _genericCopy( config.vendorfonts.src, config.vendorfonts.dest );
+  var vendorFonts = configCopy.vendorfonts;
```
Comment (sebworks) [inline gulp/tasks/copy.js]: should `vendorfonts` be camel case?

**Comment by sebworks** [comment]
👍

File: gulp/tasks/copy.js (hunk 2 above)
Comment (anselmbradford) [inline gulp/tasks/copy.js]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/2123/commits/f833fd8971ae3a992e54a515d6a994f8c0a771dc

**Comment by anselmbradford** [comment]
Fixes https://github.com/cfpb/cfgov-refresh/issues/875

**Pull request merged**